iothub/service/src/FeedbackBatch.cs

```diff
@@ -8,13 +8,28 @@ namespace Microsoft.Azure.Devices
 
     /// <summary>
     /// Contains a batch of feedback records.
+    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT Hub.
     /// </summary>
     public class FeedbackBatch
     {
+        /// <summary>
+        /// Date and time that indicates when the feedback message was received by the IoT hub.
+        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT Hub.
+        /// </summary>
         public DateTime EnqueuedTime { get; set; }
 
+        /// <summary>
+        /// A collection of feedback records of C2D messages across multiple devices in the IoT hub.
+        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT Hub.
+        /// </summary>
         public IEnumerable<FeedbackRecord> Records { get; set; }
 
+        /// <summary>
+        /// The IoT hub hostname.
+        /// </summary>
+        /// <remarks>
+        /// Despite the name of field, the value is the name of the IoT hub.
```

Comment:
Technically this is a property, not a field.

You could be generic and just say `Despite the name of this member, the value will be the name of the IoT hub.`